clang-tools-extra/clang-tidy/tool/ClangTidyMain.cpp

```diff
@@ -325,6 +325,14 @@ option is recognized.
 )"),
                                   cl::init(false), cl::cat(ClangTidyCategory));
 
+static cl::opt<bool> AllowEmptyCheckList("allow-empty-checks", desc(R"(
```

Comment:
Should the option names be in sync?
```suggestion
static cl::opt<bool> AllowEmptyCheckList("allow-empty-check-list", desc(R"(
```
or
```suggestion
static cl::opt<bool> AllowEmptyChecks("allow-empty-checks", desc(R"(
```